src/samples/audio/CMakeLists.txt

```diff
@@ -11,3 +11,9 @@ if(CONFIG_SAMPLE_KEYPHRASE)
 		detect_test.c
 	)
 endif()
+
+if(CONFIG_KWD_NN_SAMPLE_KEYPHRASE)
+        add_local_sources(sof
+                kwd_nn_detect_test.c
+        )
```

Comment:
Can this go over a single line?

Reply:
I could change it but I did so just to keep the style (please see below). Should I change the both configs?

if(CONFIG_SAMPLE_KEYPHRASE)
        add_local_sources(sof
                detect_test.c
        )
endif()

if(CONFIG_KWD_NN_SAMPLE_KEYPHRASE)
        add_local_sources(sof
                kwd_nn_detect_test.c
        )

Reply:
Everything that fits in the line should be put in one line. What is too long split, what isn't don't split.